### PR TITLE
added russian translate

### PR DESCRIPTION
**Pull request by vivazzi**
This code allow to show russian words

### CODE REVIEW

**Comment by vivazzitest** [comment]
I like this idea

**Comment by vivazzi** [comment]
Apply please!

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-This is my repo
+This is repo!!!
+
+Это репозиторий
```
Comment (vivazzitest) [inline README.md]: good!

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-This is my repo
+This is repo!!!
+
```
Comment (vivazzitest) [inline README.md]: useless blank line

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-This is my repo
+This is repo!!!
```
Comment (vivazzitest) [inline README.md]: I don't think so...

**Comment by vivazzi** [comment]
And and...

File: README.md (hunk 3 above)
Comment (vivazzitest) [inline README.md]: Okay

Comment (vivazzitest) [inline README.md]: yes

**Pull request merged**